### DISCUSSION TITLE
👩‍🌾 Sunflower Fashion Week #6

### DISCUSSION THREAD

**Discussion by BirbSunflower** [sunflower-land/sunflower-land#3196] [Art]
Welcome to the 6th Bumpkin Wearables Competition a.k.a Sunflower Fashion Week! 💃

The Art Team will choose the best outfits and integrating them into the game whenever an opportunity arises.

Before continuing, it is recommended to read the Bumpkin Guideline, which can be found here:

https://docs.sunflower-land.com/contributing/for-designers/bumpkin-guidelines

## Theme 🎨

While Sunflower Fashion Week typically embraces a diverse range of wearables without a specific theme, for this edition, we're seeking wearables that harmonize with the vibrant ambiance of the **Beach map**. 🏖

It's important to note that we've already incorporated characters like a lifeguard, a mermaid, a monkey, and fishermen, so designs should complement this eclectic cast.

Here's a submission example that suits the theme: https://github.com/sunflower-land/sunflower-land/discussions/2828#discussioncomment-6837925

## Requirements ✏️

- Dimensions of file should be 100x100px.
- The layer (e.g. shirt) should be on it's own layer without any other parts. When placed on top of a Bumpkin it should perfectly align with its body.
- Art should be original and not plagiarised.
- If submitting multiple items, an aseprite file would be appreciated to speed up review. 🤗
- By submitting art, you acknowledge you are the original creator of the art and you give Thought Farm and the community the rights to use it freely in the game and marketing purposes indefinitely. Players may trade your item without the need to pay you royalties. The community may make changes and improve artwork that is submitted.
- You agree to the [terms and services](https://docs.sunflower-land.com/contributing/competitions/bumpkin-art-competition#terms-and-conditions) of the game.
- Bumpkins live inside of the game! Please include an NPC version of your design that can be used inside of Sunflower Land.

When submitting art, please include the following:

- Your farm ID.
- A minified version of your art piece.
- An example of the art used inside of the game or on the Bumpkin. Here's a nice submission example from a community artist: https://github.com/sunflower-land/sunflower-land/discussions/2639#discussioncomment-6238691

## Rewards 🏆

The winner will receive 1000 SFL and a runner-up will receive 500 SFL, however this competition is not just about the prize money. It's an opportunity to showcase your incredible talent and become an essential part of the Sunflower Land experience. 

We want to emphasize that even if you don't win, your design may still have the chance to be selected in the future. Our team will carefully review all submissions and may reach out to designers whose work stands out for future collaboration opportunities. Sunflower Land is a community game and we love showcasing and including artwork from our players. Now is your chance to have your artwork immortalised in the #1 Community Web3 game.❤

## Deadline 📅

Designers have until **December 11th** to submit their entries. After that, our art team will carefully review and analyze each submission to select the winners.

The announcement of the winners will take place during the next week after the deadline.

We can't wait to see what amazing designs you come up with. 

Good luck to all participants! 🌻

**Comment by andand0** [comment]
### Snorkeling Guide

<br>


![snorkeling guide](https://github.com/sunflower-land/sunflower-land/assets/142132586/1099fcd8-bd1e-48b6-bd4a-3afed8bebc92)



<br>


| Snorkel | Life jacket (Coat)| Snorkel Suit| Shoes (fins)| 
|--------|--------|--------|--------|
|![snorkel](https://github.com/sunflower-land/sunflower-land/assets/142132586/7f562279-9730-4774-9876-acc13f37c46f)| ![life jacket ](https://github.com/sunflower-land/sunflower-land/assets/142132586/b97a5fac-5afc-4b1c-b3cb-e888de559bd8)|![Suit ](https://github.com/sunflower-land/sunflower-land/assets/142132586/b36f5c74-3928-4eff-b39a-e05fdc125247) | ![Fins ](https://github.com/sunflower-land/sunflower-land/assets/142132586/900d0324-e775-4503-a4de-3501aed3343d) | 



<hr>


### NPC Version


![Snorkeling-NPC-beach preview](https://github.com/sunflower-land/sunflower-land/assets/142132586/c8b73956-3ae3-4bd5-999a-dd729e6ff4e2)



| GIF 1| Sprite v1|GIF 2| Sprite V2|
|--------|--------|--------|--------|
| ![Snorkeling guide- NPC](https://github.com/sunflower-land/sunflower-land/assets/142132586/7efe14f7-9379-46d0-958a-d8fadd2c40b7) | ![Snorkeling guide- NPC](https://github.com/sunflower-land/sunflower-land/assets/142132586/4992e5f5-cd6d-4e74-9e62-e4b940dbfa1d)| ![Snorkeling guide- NPC-v2t](https://github.com/sunflower-land/sunflower-land/assets/142132586/3b0d28b0-8ab3-4df2-bab7-d703961871be) | ![Snorkeling guide- NPCv2](https://github.com/sunflower-land/sunflower-land/assets/142132586/563c8ffb-b7b3-4f16-8c99-727ba6ac7086)|



| Snorkel|Jacket| Suit| Fins|
|--------|--------|--------|--------|
|![Snorkeling guide- NPC-snorkel](https://github.com/sunflower-land/sunflower-land/assets/142132586/ed5b0426-0c8d-4096-91b7-bb5be8a75e4e) |![Snorkeling guide- NPC-jacket](https://github.com/sunflower-land/sunflower-land/assets/142132586/a4e1ba6a-4a8b-4c62-b9a6-aa714f0d980f)  |![Snorkeling guide- NPC-suit](https://github.com/sunflower-land/sunflower-land/assets/142132586/f2848348-483d-4e3e-8f68-6ee3f1ab28f1) |![Snorkeling guide- NPC-fins](https://github.com/sunflower-land/sunflower-land/assets/142132586/5239c5b4-51d4-4892-9e55-e36ebc7a77bc) | 



`Farm ID: 165810`
`Discord: Andando`

**Comment by offshoreskywalker** [comment]
Farm ID: 141648
Discord: offshoreskywalker

![walrus](https://github.com/sunflower-land/sunflower-land/assets/143333531/cb2d3c88-01d9-4a61-989b-1bc4ff965ef7)

| Walrus Tank Top | Skate Pants | Skater Shoes |
|--------|--------|--------|
| ![walrus tank top](https://github.com/sunflower-land/sunflower-land/assets/143333531/b7525447-b55e-47f3-baa7-2e79983fff10) | ![skate pants](https://github.com/sunflower-land/sunflower-land/assets/143333531/3020bd44-e8ae-4923-8e8e-e56e547fe0c4) | ![skater shoes](https://github.com/sunflower-land/sunflower-land/assets/143333531/2ef46e62-78d0-4fd4-9aae-6726321a1be9) |


![ducky](https://github.com/sunflower-land/sunflower-land/assets/143333531/f51e0a02-9cc0-4a49-babe-3c5ebbd19bae)

Visor | Ducky Tank Top | Swim Shorts | Flip Flops | 
|--------|--------|--------|--------|
| ![visor](https://github.com/sunflower-land/sunflower-land/assets/143333531/2127082d-7931-458c-865e-6b132cdef1a1) | ![ducky tank top](https://github.com/sunflower-land/sunflower-land/assets/143333531/b845687b-0d45-4f80-8a7d-6f7fc940da6a) | ![swim shorts](https://github.com/sunflower-land/sunflower-land/assets/143333531/c9d66ff9-dcc9-4eae-9109-61fbf8d59f26) | ![flipflops](https://github.com/sunflower-land/sunflower-land/assets/143333531/d4a6fcd6-d829-4fd6-b6ca-e42410a28fef) |


![Kraken Wetsuit](https://github.com/sunflower-land/sunflower-land/assets/143333531/32e2625f-bc31-4404-bc36-366b97feeb57)

Kraken Wetsuit | Water Shoes | 
|--------|--------|
| ![wetsuit](https://github.com/sunflower-land/sunflower-land/assets/143333531/1bd93bcb-b332-4970-975b-d7327af0394b) | ![water shoes](https://github.com/sunflower-land/sunflower-land/assets/143333531/2cbb939e-1ab5-4a1b-83a1-29596e6adcc9) |

**Comment by peperol** [comment]
Beach party girl
![In spiaggia -1](https://github.com/sunflower-land/sunflower-land/assets/114354077/5666c4a4-71f6-4556-8a41-908ec1e818d4)

coconut bra
![Coconut Bra](https://github.com/sunflower-land/sunflower-land/assets/114354077/5973a891-7b0e-4f9b-b533-75b3ff88cd19)
bananas skirt
![Skirt banana](https://github.com/sunflower-land/sunflower-land/assets/114354077/d4e542bd-8408-48dc-a038-876e7704a4d3)
inflatable strawberry
![Gonfiabile92-1](https://github.com/sunflower-land/sunflower-land/assets/114354077/e17daefd-5623-489c-86a9-471eb58bde02)
fishbone comb
![Pettine fish bones ](https://github.com/sunflower-land/sunflower-land/assets/114354077/780cae8a-55d9-4fa3-89a8-e367d851e411)

scale mermaid necklace
![Scale mermaid necklace 92-1](https://github.com/sunflower-land/sunflower-land/assets/114354077/877cab3e-11f8-4e06-939d-60df605ace90)

npc
![Banananpc7](https://github.com/sunflower-land/sunflower-land/assets/114354077/39577cea-839f-46a8-b445-0c74054213a3)

in land
![In Land ](https://github.com/sunflower-land/sunflower-land/assets/114354077/8b0055cc-8df4-4ce5-beaf-41f498dc4433)




Pirata girl
![pirate woman](https://github.com/sunflower-land/sunflower-land/assets/114354077/419d0494-053c-46c6-a52e-c8f8a9d5b893)

pirata top
![pirate top](https://github.com/sunflower-land/sunflower-land/assets/114354077/0cb2635c-2dfd-40e6-bb74-e73fd785b275)
pirate skirt
![pirate skirt](https://github.com/sunflower-land/sunflower-land/assets/114354077/96e23415-0219-40ad-912b-5b66ded72ca9)
pirate badana
![pirate bandana](https://github.com/sunflower-land/sunflower-land/assets/114354077/fa649a2b-d599-44c3-b131-045367226c0e)
pirate shoes
![pirate woman shoes](https://github.com/sunflower-land/sunflower-land/assets/114354077/75185b15-8b02-4fa7-af1a-d851b475501e)
pirata pepita necklace
![pepita neklace](https://github.com/sunflower-land/sunflower-land/assets/114354077/7406c4db-2bb2-4828-b4b5-c74126aa7698)
pirate gun
![pirate gun](https://github.com/sunflower-land/sunflower-land/assets/114354077/eca7f8dc-9830-4993-894b-ec100a8565ca)

npc
![Gif5](https://github.com/sunflower-land/sunflower-land/assets/114354077/425fd89f-9117-41dd-8642-f83a1b96e229)

in land
![Pirata 300lo](https://github.com/sunflower-land/sunflower-land/assets/114354077/2cc4a579-c772-4a99-911c-315e3d212c7d)

Castaway

![Opera_senza_titolo-2](https://github.com/sunflower-land/sunflower-land/assets/114354077/6a0e51a1-1b96-4dc5-b43f-6bbb3b9741ad)




Hair

![Jairo 1](https://github.com/sunflower-land/sunflower-land/assets/114354077/bcbcaa2e-8953-4f3d-af87-7ccd0464686a)

Faded and torn T-shirt

![T-shirt 297-1](https://github.com/sunflower-land/sunflower-land/assets/114354077/e39cd8cb-61a2-4bb2-ad32-44cf3d449d1d)

pants
![Pantaloni-1](https://github.com/sunflower-land/sunflower-land/assets/114354077/d4be39e8-b85e-420d-8ca6-dba43a74fd10)

Friend “Sunny”

![Palloneok-1](https://github.com/sunflower-land/sunflower-land/assets/114354077/1599a80b-f45b-482a-aeeb-47accc6f7532)

Necklace

![Necklace analisi 1](https://github.com/sunflower-land/sunflower-land/assets/114354077/417c6536-652c-49be-962c-87b6c3ad648d)
)

NPC

![Gif ok0318](https://github.com/sunflower-land/sunflower-land/assets/114354077/41638bef-5f44-4dbc-b8ec-7435094c5139)

In Land

![In Land tolo](https://github.com/sunflower-land/sunflower-land/assets/114354077/21ba1caa-7c9c-4914-9ff2-06866ddb85ec)


farm ID 145849
discord peperol

**Comment by Vergelsxtn** [comment]
### Chilling Bumpkin 🏝🎶

**Bumpkin** 
![prev](https://github.com/sunflower-land/sunflower-land/assets/99605039/971aaa90-704e-4bac-a244-343b3f8d3bc9)
![prev1](https://github.com/sunflower-land/sunflower-land/assets/99605039/c143ccc3-cb85-4509-9c5e-f46a6ceda9dd)
(both human and goblin body compatible)

| hatwig | guitar | beard | shirt | pants | glasses |
|--------|--------|--------|--------|--------|--------|
| ![hatwig](https://github.com/sunflower-land/sunflower-land/assets/99605039/a5ff42e1-41fb-46ef-94a8-8593591b256c) | ![guitar](https://github.com/sunflower-land/sunflower-land/assets/99605039/cf7f0242-1f86-4a6a-b76f-4f8c7edfcc66) | ![beard](https://github.com/sunflower-land/sunflower-land/assets/99605039/07c6e588-8399-4f1c-a4a6-067d8d1f1b26) | ![shirt](https://github.com/sunflower-land/sunflower-land/assets/99605039/217fc103-b105-431a-b420-1b5ee25c376f) | ![pants](https://github.com/sunflower-land/sunflower-land/assets/99605039/5c10d8e6-d6ec-40cb-9012-10c04051a530) | ![glasses](https://github.com/sunflower-land/sunflower-land/assets/99605039/bd6e1960-e40c-43ff-9464-115d57d71326) |

NPC
![chilling-bumpkin-npc1](https://github.com/sunflower-land/sunflower-land/assets/99605039/0380449a-4ede-47fd-b7b4-d62d49e12492)

| spritesheet | |
|--------|--------|
|hatwig|![hatwig](https://github.com/sunflower-land/sunflower-land/assets/99605039/ae354ba7-f304-4653-9169-1df90073dcd6)|
|beard|![beard](https://github.com/sunflower-land/sunflower-land/assets/99605039/d13f8ce9-e6b5-47d4-8ac3-742c00fe0fad)|
|shirt|![shirt](https://github.com/sunflower-land/sunflower-land/assets/99605039/205fa949-7de6-4d2e-855b-777bf8de5dbc)|
|pants|![pants](https://github.com/sunflower-land/sunflower-land/assets/99605039/baaf8128-6736-4b9f-ac1d-5cc1574da0b5)|
|glasses|![glasses](https://github.com/sunflower-land/sunflower-land/assets/99605039/94ab393f-1fa0-4b90-b122-89d1abdc7ef3)|

Farm ID: 63

❤

**Reply by jorayy** [reply]
Nice
